blockdag/dag.go

```diff
@@ -1472,44 +1472,44 @@ func (dag *BlockDAG) IsInSelectedParentChain(blockHash *daghash.Hash) bool {
 	return dag.virtual.selectedParentChainSet.containsHash(blockHash)
 }
 
-// SelectedParentChain returns the selected parent chain starting from startHash (exclusive)
-// up to the virtual (exclusive). If startHash is nil then the genesis block is used. If
-// startHash is not within the select parent chain, go down its own selected parent chain,
+// SelectedParentChain returns the selected parent chain starting from lowHash (exclusive)
+// up to the virtual (exclusive). If lowHash is nil then the genesis block is used. If
+// lowHash is not within the select parent chain, go down its own selected parent chain,
 // while collecting each block hash in removedChainHashes, until reaching a block within
 // the main selected parent chain.
 //
 // This method MUST be called with the DAG lock held
-func (dag *BlockDAG) SelectedParentChain(startHash *daghash.Hash) ([]*daghash.Hash, []*daghash.Hash, error) {
-	if startHash == nil {
-		startHash = dag.genesis.hash
+func (dag *BlockDAG) SelectedParentChain(lowHash *daghash.Hash) ([]*daghash.Hash, []*daghash.Hash, error) {
```

Comment:
I think you can still use startHash here, or at least change it to blockHash. I don't like the name lowHash without a correspondent highHash.